### PR TITLE
Show "swipe for more" only once

### LINKED ISSUE #7681: "Swipe for more" message should show once
The "Swipe for more" message we show in reader detail & notification detail is annoying. Right now we don't tell it to stop appearing until the user swipes. Instead, we should show it once and be done with it.

### PR DESCRIPTION
**Pull request by nbradbury**
Fixes #7681 - changes the reader detail and notification detail to show the "swipe for more" message only once. Prior to this PR, the message would continue to appear until the user performed a swipe.

screenshot_1527635986

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java [hunk 1]
```diff
@@ -708,6 +705,7 @@ public void run() {
                         // let the user know they can swipe between posts
                         if (adapter.getCount() > 1 && !AppPrefs.isReaderSwipeToNavigateShown()) {
```
Comment (khaykov) [inline WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java]: This is not directly related to this issue, but when I open any post other then first, `adapter.getCount()` returns 1 and the toast is not shown 🤔

Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java]: `NotificationsDetailActivity` works as expected.

Comment (nbradbury) [reply WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java]: @khaykov Sorry, I only just now realized you commented here! I'm not able to reproduce the issue with `getCount()` returning 1, and looking at the code I don't see how it's possible unless there really is only one item. Could you tell me the steps to repro that?

Comment (khaykov) [reply WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java]: Huh. It works as expected now :) Sorry about that! I blame 👻
